checker/build.gradle

```diff
@@ -1085,6 +1093,39 @@ task wpiPlumeLibTest(group: 'Verification') {
     }
 }
 
+// Unzip JSpecify conformance tests using the artifact
+task unzipConformanceTestSuite(group: 'Verification') {
```

Comment:
Please add `JSpecify` to these tasks and their descriptions.

Reply:
Should this task have a `group`? Tasks with a group are shown in `gradlew tasks`. Is this task ever directly useful?